### ISSUE TITLE
why this flatpak app doesn't use libtorrent-v2?

### ISSUE THREAD

**Issue by rezad1393** [flathub/org.qbittorrent.qBittorrent#132]
why this flatpak app doesn't use libtorrent-v2?

I can't use this app for hybrid or v2 torrents.

**Comment by zefr0x** [comment]
It used to provide libtorrent-v2, tell this commit: [#cc30f883d8eca2372b2368b57c70bbba06d86649](https://github.com/flathub/org.qbittorrent.qBittorrent/commit/cc30f883d8eca2372b2368b57c70bbba06d86649)

Was there any problem in qbittorrent v4.x? Was it fixed by now?

**Comment by Chocobo1** [comment]
>Was there any problem in qbittorrent v4.x? Was it fixed by now?

No. There are out-of-memory and stability issues with libtorrent v2 and users are reporting better performance with libtorrent v1.
Search about it in qbittorrent issue tracker: https://github.com/qbittorrent/qBittorrent/issues

**Comment by tibequadorian** [comment]
Sorry, but what to search for exactly? There are ~2.8k issues. Please link those issues so it can be kept track of.

**Comment by luzpaz** [comment]
@tibequadorian https://github.com/qbittorrent/qBittorrent/issues?q=is%3Aissue+is%3Aopen+label%3ALibtorrent+sort%3Acomments-desc

Notably: 
https://github.com/qbittorrent/qBittorrent/issues/17489
https://github.com/qbittorrent/qBittorrent/issues/17082  
https://github.com/qbittorrent/qBittorrent/issues/9782

**Comment by lucasmz-dev** [comment]
huh... i actually am looking for v2 because i'm having ram issues with v1. oh well.

it sucks that there's no way to have qBit v2 on flatpak, having trouble finding anything that supports v2, and it's harder when you want to also run your client in a sandbox :)

**Comment by nihil-admirari** [comment]
I haven't checked it myself, but [Fedora Flatpaks](https://docs.fedoraproject.org/en-US/flatpak/installation/) probably ship qBittorrent compiled with libtorrent-v2.

**Comment by koloved** [comment]
thank you for letting me know, yes its there, seems like its time to switch to lb2 for main flatpack - 

`flatpak remote-add --title "Fedora Flatpaks" fedora oci+https://registry.fedoraproject.org`
`flatpak install fedora org.qbittorrent.qBittorrent`

<img width="851" height="571" alt="Image" src="https://github.com/user-attachments/assets/a24f338d-c511-4cc2-8eda-62ac1eef10ba" />
